Authentication/AuthenticationBootstrapper.swift

```diff
@@ -12,8 +12,9 @@ import Foundation
     Bootstrapper to start the application.
     Takes care of starting the authentication process before the main View Controller of the app when necessary,
     and after the user logs out.
+    The authentication process includes login, signup and recover password logic.
 */
-public class AuthenticationBootstraper<User: UserType, SessionService: SessionServiceType where SessionService.User == User> {
+public class AuthenticationBootstrapper<User: UserType, SessionService: SessionServiceType where SessionService.User == User> {
```

Comment:
We need to add an extension point (a method) to be able to set the view delegate without having to override the `createLoginView` method. Maybe the `ApplicationBootstrapper` could have a `var loginViewDelegate: LoginViewDelegate? = .None` property. What do you think?